src/config/index.js

```diff
@@ -252,10 +263,11 @@ function Config() {
         config.inventory.host = getHostForApp(dependencyEndpoints, 'host-inventory', 'service', 'INVENTORY') || env.INVENTORY_HOST || 'http://insights-inventory.platform-ci.svc.cluster.local:8080';
         config.patchman.host = getHostForApp(dependencyEndpoints, 'patchman', 'manager', 'PATCHMAN') || env.PATCHMAN_HOST || 'http://localhost:8080';
         config.rbac.host = getHostForApp(dependencyEndpoints, 'rbac', 'service', 'RBAC') || env.RBAC_HOST || 'http://localhost:8080';
+        config.kessel.url = getHostForApp(dependencyEndpoints, 'kessel-relations', 'api', 'KESSEL_URL') || env.KESSEL_URL || 'http://localhost:8080';
```

Comment:
This should be setup already above and kessel is not being reported correctly by clowder for now

```suggestion
```